runatlantis.io/docs/using-atlantis.md

```diff
@@ -124,6 +124,36 @@ Because Atlantis under the hood is running `terraform apply plan.tfplan`, any Te
 They're ignored because they can't be specified for an already generated planfile.
 If you would like to specify these flags, do it while running `atlantis plan`.
 
+---
+## atlantis import
+
+```bash
+atlantis import [options] ADDRESS ID -- [terraform import flags]
+```
+### Explanation
+Runs `terraform import` that matches the directory/project/workspace.
+This command discards terraform plan result. Before apply, required `atlantis plan` again.
```

Comment:
```suggestion
This command discards the terraform plan result. After an import and before an apply, another `atlantis plan` must be run again.
```

Reply:
Because of the discarding of the plan after an import, should there be another flag to auto plan a project/dir/workspace after a successful import?

Also if an import command fails can we retain the plan instead of discarding it?

Reply:
> Because of the discarding of the plan after an import, should there be another flag to auto plan a project/dir/workspace after a successful import?

Currently, there is not such flag.
I thought that if users want to import one resource, autoplan is useful. But if try to import multiple imports, autoplan make the operation slow.
`terraform import` do not plan again, so I implements no-autoplan by default. We can provide such flag in the future as optional.

> Also if an import command fails can we retain the plan instead of discarding it?

import_step_runner discard planfile if import has no err 👌 

```go
	out, err := p.TerraformExecutor.RunCommandWithVersion(ctx, filepath.Clean(path), importCmd, envs, tfVersion, ctx.Workspace)

	// If the import was successful and a plan file exists, delete the plan.
	planPath := filepath.Join(path, GetPlanFilename(ctx.Workspace, ctx.ProjectName))
	if err == nil {
```

Reply:
Perhaps we can have an optional subcommand like we do for `atlantis apply` e.g. `--auto-merge-disabled`

For this case, perhaps a flag for `--autoplan`, `--autoplan-on-success` ? This could be added in a separate PR.

Reply:
`--autoplan` looks good for import option. We can do `atlantis pllan` manually, so it is better to implement this after raised issue got votes.